vagrant/provisioning/roles/elasticsearch-install/templates/elasticsearch.yml

```diff
@@ -0,0 +1,14 @@
+cluster.name: {{ cluster_name | default('acm-cluster') }}
```

Comment:
@bojmil-armedia , @fabricetriboix , do we prefer generic 'cluster_name' and 'node_name' here, or would we rather have 'elastic_cluster_name' and 'elastic_node_name'

Reply:
It is more descriptive and specific with **elastic** prefix. Or **elasticsearch** as the other variables.